MAUI/Maps/migration.md

```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
+    
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapShapeLayer ShapesSource="https://cdn.syncfusion.com/maps/map-data/world-map.json">
+            <map:MapShapeLayer.Sublayers>
+                <map:MapArcLayer>
+                    <map:MapArcLayer.Arcs>
+                        <map:MapArc StrokeThickness="2"
+                                    Stroke="#8a8a8a" 
+                                    ControlPointFactor="0.5" 
+                                    HeightFactor="0.2">
+                            <map:MapArc.From>
+                                <map:MapLatLng Latitude="77.1025"
+                                               Longitude="28.7041" />
+                            </map:MapArc.From>
+                            <map:MapArc.To>
+                                <map:MapLatLng Latitude="-106.3468"
+                                               Longitude="56.1304" />
+                            </map:MapArc.To>
+                        </map:MapArc>
+                    </map:MapArcLayer.Arcs>
+                </map:MapArcLayer>
+            </map:MapShapeLayer.Sublayers>
+        </map:MapShapeLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+ public MainPage()
+{
+    InitializeComponent();
+    SfMaps maps = new SfMaps();
+    MapShapeLayer layer = new MapShapeLayer();
+    layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/world-map.json"));
+    MapArcLayer mapArcLayer = new MapArcLayer();
+    MapArc arc = new MapArc();
+    arc.From = new MapLatLng(77.1025, 28.7041);
+    arc.To = new MapLatLng(-106.3468, 56.1304);
+    arc.Stroke = Color.FromRgb(61, 155, 242);
+    arc.StrokeThickness = 2;
+    arc.ControlPointFactor = 0.5;
+    arc.HeightFactor = 0.2;
+    layer.Sublayers.Add(mapArcLayer);
+    maps.Layer = layer;
+    this.Content = maps;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
+### Circle Layer
+
+<table>
+<tr>
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'MapCircle'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Center'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Radius'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Stroke'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeThickness'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Fill'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```

Comment:
The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

Reply:
Modified